internal/conf/configuration.go

```diff
@@ -111,18 +111,22 @@ type JWTConfiguration struct {
 }
 
 type MFAFactorTypeConfiguration struct {
+	EnrollEnabled bool `json:"enroll_enabled" split_words:"true" default:"false"`
```

Comment:
Changed this to default to false for use with MFA (Phone) and MFA (WebAuthn). We then handle MFA (TOTP) as a special case